assets/constants.js

```diff
@@ -609,6 +609,12 @@ var constants = {
   // eslint-disable-next-line max-len
   "DEFAULT_TWITTER_SHARE_MESSAGE_EDITOR": "Check out this interactive lesson I created on Oppia - a free platform for teaching and learning!",
 
+  // eslint-disable-next-line max-len
+  "DEFAULT_SOLICIT_ANSWER_DETAILS_QUESTION_HTML": "<p translate='I18N_SOLICIT_ANSWER_DETAILS_QUESTION'></p>",
```

Comment:
As mentioned previously, no HTML in JS please. constants.js is still JS.

Reply:
Done